### DISCUSSION TITLE
How to correctly transfer an html page

### DISCUSSION THREAD

**Discussion by valendovsky** [uNetworking/uWebSockets#1293] [Q&A]
I am using uWS :: App(). Ws <PerSocketData>. When I connect through the local html page, there is a connection. But when I send the page through the get (" / * ", [] (auto * res, auto * req) {res->end(ss.str().c_str());}), JavaScript don't work. I would be grateful if you can tell me what is the error?

**Comment by ghost** [comment]
You can compare the headers and content in Chrome, with headers and content from some other server and make the changes needed. Maybe you're missing content type or something

**Comment by valendovsky** [comment]
The problem has been resolved.
Previously, I read the page into a variable of type std::stringstream, then translated it into a C-style string and sent it to the client. The script did not work adequately. I began to read the page into a variable of type std::string and all the script began to work.
Thank you for the nice library.
